include/ipc/rpmsg_multi_instance.h

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_RPMSG_MULTIPLE_INNSTANCE_H_
+#define ZZEPHYR_INCLUDE_RPMSG_MULTIPLE_INNSTANCE_H
+
+#include <openamp/open_amp.h>
+#include <metal/sys.h>
+#include <metal/device.h>
+#include <metal/alloc.h>
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief RPMsg multiple instance API
+ * @defgroup rpmsg_service_api RPMsg multiple instance APIs
+ * @{
+ */
+
+
+#define VDEV_START_ADDR     CONFIG_RPMSG_MULTI_INSTANCE_SHM_BASE_ADDRESS
+#define VDEV_SIZE           CONFIG_RPMSG_MULTI_INSTANCE_SHM_SIZE
+
+#define SHM_START_ADDR      VDEV_START_ADDR
+#define SHM_SIZE            VDEV_SIZE
+
+#define VRING_ALIGNMENT     (4)     /**< Alignment of vring buffer.*/
+#define VDEV_STATUS_SIZE    (0x4)   /**< Size of status region. */
+
+
+/** @brief Event callback structure
+ *  It is regisered during endpoint registration.
+ *  This structure is packed into the endpoint configuration.
+ */
+struct rpmsg_mi_cb {
+	/** @brief Bind was successful
+	 *  @param priv Private user data
+	 */
+	void (*bound)(void *priv);
+
+	/** @brief New packet arrived
+	 *  @param data Pointer to data buffer
+	 *  @param len Length of @a data
+	 *  @param priv Private user data
```

Comment:
Just for consistency, add full stop to the end of each parameter description.

Reply:
Done.